xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingService.java

```diff
@@ -224,6 +245,88 @@ protected WebSocketClientHandler getWebSocketClientHandler(
 
   private WebSocketClientHandler.WebSocketMessageHandler channelInactiveHandler = null;
 
+  String submitLimitOrder(KrakenStreamingLimitOrder krakenStreamingLimitOrder) {
```

Comment:
sorry missed these the first time through but the timeouts how did you decide on 60s for sending a limit order and 1s for canceling? Should the latter be a little longer?

Reply:
Kraken has some issues and sometimes you don't get any response for quite a long time. In case this is about placing a new order - no drama - at least in my case. In case it is about cancelling an order - I want to get the control quickly back to issue cancel via REST.